src/main/java/com/google/sps/data/PossibleInterviewSlot.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.auto.value.AutoValue;
+
+/**
+ * A PossibleInterviewSlot represents a 1 hour chunk of time with a utcEncoding and readable date
+ * and time strings.
+ */
+@AutoValue
+public abstract class PossibleInterviewSlot {
+  public abstract String utcEncoding();
```

Comment:
Is this an encoded timestamp in utc?

UTC encoding sounds like "the description of how a utc timestamp is encoded", not "an encoded timestamp"

Reply:
I can see that there's a bunch of existing code that uses this unfortunate verbiage, so it might be onerous to fix this.

Oh well.